spec/intl.html

```diff
@@ -1338,6 +1338,259 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+              _duration_: a %Temporal.Duration%,
+              _overflow_: an Object,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to add _duration_ to _date_ in the context of the calendar represented by the string _calendar_ and returns the corresponding day, month and year of the result in the ISO calendar values as a Record.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedifference" type="abstract operation">
+          <h1>
+            CalendarDateDifference (
+              _calendar_: a String,
+              _one_: a %Temporal.PlainDate%,
+              _two_: a %Temporal.PlainDate%,
+              _largestUnit_: a String,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to find the difference between the two dates _one_ and _two_ in the context of the calendar represented by the string _calendar_ and returns the corresponding years, months, weeks and days as a Record.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateera" type="abstract operation">
+          <h1>
+            CalendarDateEra (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to find the era for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateerayear" type="abstract operation">
+          <h1>
+            CalendarDateEraYear (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to find the year of the current era for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateyear" type="abstract operation">
+          <h1>
+            CalendarDateYear (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to find the year for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</dd>
+          </dl>
```

Comment:
(Copying from and adapting my earlier comment on the previous text) Somewhere (either here or elsewhere) we need to define expectations for what `year` is.

Here's one attempt at this:  

_Returns a signed integer relative to the first day of a calendar-specific "epoch year", which is typically the first year of a particular era for calendars that use eras, or is year 1 or 0 for calendars that don't use eras.  For example, the `'gregory'` calendar's epoch year is 1 CE, so `year` for 3 CE should be `3`, and `year` for 2 BCE should be `-1`._  

_Note that `year` is relative to the first day of the calendar's epoch year, so if the epoch era starts in the middle of the year, the `year` will be the same value before and after the start date of the era. This is different from `eraYear`, which will change when a new era starts mid-year._

Somewhere (probably not here?) we need to specify for each ICE calendar which era is used as the epoch era and what the epoch date is in both calendar and ISO dates. It would be bad if different implementations picked different epoch dates!